src/DeterministicVI.jl

```diff
@@ -16,16 +16,23 @@ export ElboArgs
 
 """
 ElboArgs stores the arguments needed to evaluate the variational objective
-function
+function.
 """
 type ElboArgs{NumType <: Number}
     S::Int64
     N::Int64
+
+    # The number of components in the point spread function.
+    psf_K::Int64
     images::Vector{TiledImage}
     vp::VariationalParams{NumType}
     tile_source_map::Vector{Matrix{Vector{Int}}}
     patches::Matrix{SkyPatch}
     active_sources::Vector{Int}
+
+    # The number of standard deviations away below which a bivariate normal
+    # will be calculated.  Set to Inf to evaluate bivariate normals at all points.
+    num_allowed_sd::Float64
```

Comment:
Are you sure two standard deviations is enough? Back when I was using `NLopt` I had serious convergence problems with discontinuities in the objective created by dynamically truncating the PSF. Now, NLopt was not particularly robust, but nonetheless, this seems really risk too me. If it works, it only works for a particular optimizer, not optimizers in general. Some are very sensitive to discontinuities. Why would we voluntarily introduce discontinuities? What's wrong with fixing the active pixels a priori?